docs/groovy-script/mods/mwc/crafting_station.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Crafting Station/Ammo Press"
+titleTemplate: "Modern Warfare Cubed | CleanroomMC"
+description: "Converts multiple items into another item, with a way to recover some of original resources when dismantled."
+source_code_link: "https://github.com/Cubed-Development/Modern-Warfare-Cubed/blob/next/src/main/java/com/paneedah/mwc/groovyscript/script/CraftingStation.java"
+---
+
+# Crafting Station/Ammo Press (Modern Warfare Cubed)
+
+## Description
+
+Converts multiple items into another item, with a way to recover some of original resources when dismantled.
+
+## Identifier
+
+Refer to this via any of the following:
+
+```groovy:no-line-numbers {1}
+mods.mwc.crafting_station/* Used as page default */ // [!code focus]
```

Comment:
might want to add more aliases - probably want `Alias.generateOfClassAnd(CraftingStation.class, "AmmoPress")`?